configure.ac

```diff
@@ -254,7 +254,7 @@ AC_CHECK_TOOL(CPPO,cppo)
 AC_CHECK_TOOL(PATCH,patch)
 AC_CHECK_TOOL(BUNZIP2,bunzip2)
 
-AX_COMPARE_VERSION([$OCAMLVERSION], [lt], [4.07.0],
+AX_COMPARE_VERSION([$OCAMLVERSION], [lt], [4.08.0],
```

Comment:
Ouch (my mistake...) 😕

Reply:
`configure` not updated accordingly (as for #4592). Shouldn't CI error in that case?

Reply:
Eek: https://github.com/ocaml/opam/pull/4587/checks?check_run_id=2056831138#step:4:48

Reply:
ok...